ale/base/data_naif.py

```diff
@@ -523,8 +525,11 @@ def sensor_position(self):
                               "frame": "J2000",
                               "abcorr": self.light_time_correction,
                               "mission": self.spiceql_mission,
-                              "searchKernels": self.search_kernels}
-                    obs_tars = self.spiceql_call("getTargetStates", web=self.use_web, function_args=kwargs)
+                              "searchKernels": self.search_kernels,
+                              "limitCk": -1, 
+                              "limitSpk" : 1,
```

Comment:
These are due to them not being interpreted correctly for some reason. Might be a bug with how SpiceQL web is handling default params?